### PR TITLE
ICU-21957 BRS72RC Update version number and regenerate configure v72.1

### PR DESCRIPTION
**Pull request by rp9-next**
##### Checklist

- [x] Required: Issue filed: https://unicode-org.atlassian.net/browse/ICU-21957
- [x] Required: The PR title must be prefixed with a JIRA Issue number. 
- [x] Required: The PR description must include the link to the Jira Issue, for example by completing the URL in the first checklist item
- [x] Required: Each commit message must be prefixed with a JIRA Issue number. 
- [x] Issue accepted (done by Technical Committee after discussion)
- [ ] Tests included, if applicable
- [ ] API docs and/or User Guide docs changed or added, if applicable

### CODE REVIEW

File: icu4c/source/configure [hunk 1]
```diff
@@ -4264,6 +4264,8 @@ fi
 #AC_CHECK_PROG(AUTOCONF, autoconf, autoconf, true)
 #AC_CHECK_PROG(STRIP, strip, strip, true)
 
+
+
```
Comment (daniel-ju) [inline icu4c/source/configure:4268]: nit: extra whitespace

**Review by pedberg-icu — APPROVED** [review]

**Comment by markusicu** [comment]
FYI  For ICU pull requests, we set the assignee to the main reviewer, not to the PR author. The PR author is obviously doing the work already :-)
See https://icu.unicode.org/repository/gitdev

File: icu4c/source/configure (hunk 1 above)
Comment (markusicu) [reply icu4c/source/configure:4268]: This file is generated, so I assume that we don't have control over the formatting.

**Review by markusicu — APPROVED** [review]
lgtm tnx

**Review by daniel-ju — APPROVED** [review]

**Pull request merged**